lib/HunkView.ml

```diff
@@ -31,44 +110,54 @@ let ui_hunk_summary (hunk : Patch.hunk) : Nottui.ui =
       at_symbols;
     ]
 
-let rec process_lines (mine_num : int) (their_num : int) (acc : Nottui.ui list)
-    = function
+let process_line (hunk : Patch.hunk) i line =
+  let line_number =
+    W.string
+      ~attr:Notty.A.(fg yellow)
+      (Printf.sprintf "%4d " (succ (hunk.Patch.mine_start + i)))
+  in
+  match line with
+  | Common s ->
+      let content = W.string s in
+      Ui.hcat [ line_number; content ]
```

Comment:
Previously, this function returned `(line_num_mine, line_num_their, ui)`, which was better as it could be shared with the split view.
Implementing word-diff in the split-view (in a later PR) will cause duplication due to this.